Task2.3/Classes/Trainig.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2._3.Classes
+{
+    public class Trainig
+    {
```

Comment:
private field without any chances to set it?)))

Reply:
That supposed to be a property... but ctrl+c...